src/Apps/Order/Components/PartnerOfferTimerItem.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react"
+import { useCountdownTimer } from "Utils/Hooks/useCountdownTimer"
+import StopwatchIcon from "@artsy/icons/StopwatchIcon"
+import { DateTime } from "luxon"
+import { Box, ProgressBar, Spacer, Text } from "@artsy/palette"
+import { graphql, useFragment } from "react-relay"
+import { PartnerOfferTimerItem_order$key } from "__generated__/PartnerOfferTimerItem_order.graphql"
+
+export const PartnerOfferTimerItem: React.FC<{
+  order: PartnerOfferTimerItem_order$key
+}> = ({ order }) => {
+  const data = useFragment(query, order)
+
+  const startTime = data.stateUpdatedAt || ""
+  const endTime = data.stateExpiresAt || ""
+
+  const { remainingTime, percentComplete } = useCountdownTimer({
+    startTime: startTime,
+    endTime: endTime,
+  })
+
+  if (remainingTime.match(/^NaN/)) {
+    return null
+  }
+
+  if (data.displayState !== "PENDING") {
+    return null
+  }
+
+  const actionDeadline = DateTime.fromISO(endTime, {
+    zone: "America/New_York",
+  }).toFormat("MMM d, h:mm a ZZZZ")
+
+  return (
+    <>
+      <Box flexDirection="row" textAlign="center">
+        <>
```

Comment:
Super minor, but we shouldn't need this fragment as everything is being contained inside `Box`.

Reply:
Thanks for the comment. True, I didn't need that fragment. Removed it.